src/routes/test/+page.svelte

```diff
@@ -2767,9 +2767,24 @@
 			</div>
 		</div>
 	</section>
+	<div class="sidebar vertical">
+		<Button>Test</Button>
+	</div>
 </div>
 
 <style>
+	.sidebar {
+		border-right: 1px solid var(--colors-low);
+		padding: var(--padding);
+		width: 320px;
+		height: 100%;
```

Comment:
This does nothing. Instead I'd suggest to put in some fixed height, e.g.
```suggestion
		height: 200px;
```